pkg/jsonlog/jsonlog_marshalling.go

```diff
@@ -101,6 +101,15 @@ func (mj *JSONLog) MarshalJSONBuf(buf *bytes.Buffer) error {
 		buf.WriteString(`"stream":`)
 		ffjsonWriteJSONString(buf, mj.Stream)
 	}
+	if len(mj.Tag) > 0 {
+		if first {
+			first = false
+		} else {
+			buf.WriteString(`,`)
+		}
+		buf.WriteString(`"tag":`)
+		ffjsonWriteJSONString(buf, mj.Tag)
+	}
```

Comment:
Did you use `ffjson` to generate this code? According to the comment on this file that is how it was done originally.

Reply:
No, I skipped to where it broke and fixed it thereby missing the comment at the top that I did not anticipate. I'll work on applying the ffjson generation code as well as the patch that is in the comment section at the top of the file.

Reply:
I followed the process documented in the file. I ran the command 'go get -u github.com/pquerna/ffjson' and then 'ffjson pkg/jsonlog/jsonlog.go'. Upon opening the new file (jsonlog_ffjson.go), the contents look nothing like the original source. 

I suspect the ffjson tool has been updated, but it is substantially different. I see an unmarshal function, but not a marshal function. I have the new output from ffjson over at https://gist.github.com/bonczj/6f683afbc888c8d517601d79c6a7acb7.

Reply:
Ok, I looked into this and opened #34946 to fix the comment. Don't worry about generating the code from ffjson, what you have will work for now.